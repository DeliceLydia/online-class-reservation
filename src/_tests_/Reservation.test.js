import { BrowserRouter, Route } from 'react-router-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import createStore from '../redux/createStore';
import Reservation from '../containers/ReservationsList';

const store = createStore();

describe('Reservations', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Provider store={store}>
            <Route
              path="/"
              component={(props) => <Reservation history={props.history} />}
            />
          </Provider>
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
