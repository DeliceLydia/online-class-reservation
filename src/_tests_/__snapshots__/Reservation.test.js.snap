// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservations should render correctly 1`] = `
Array [
  <nav
    className=""
  >
    <div
      className="flex navbar align"
    >
      <div
        className="nav-left"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <span
            className="logo"
          >
            0nlineCourses
          </span>
        </a>
      </div>
      <div
        className="nav-right align"
      >
        <a
          className="nav login"
          href="/teachers"
          onClick={[Function]}
        >
          Teachers
        </a>
        <a
          className="nav signup"
          href="/logout"
          onClick={[Function]}
        >
          Logout
        </a>
      </div>
    </div>
  </nav>,
  "List is empty",
]
`;
